chart-templates/crd-base/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: ${api_version}
+version: ${chart_version}
+description: A Rancher chart that creates ${name} CRDs within a cluster.
+name: ${name}-crd
+type: application
```

Comment:
Depending on which PR gets merged in first, we may need to update the `Chart.yaml` here based on https://github.com/rancher/charts/pull/604#discussion_r483348846.

Reply:
We can do this now.

Reply:
Updated in latest push